docs/source/requirements.txt

```diff
@@ -0,0 +1,25 @@
+codecov
+coverage
+flake8
+joblib
+loky
+matplotlib
+mlinsights
+mlprodict
+myst-parser
+nbsphinx
+onnx
+pillow
+py-spy
+pandas
+pyinstrument
+pyquickhelper>=1.9.3359
+pytest
+pytest-cov
+scikit-learn
```

Comment:
it shoud work without codecov, coverage, flake8, joblib, loky, py-spy, pyinstrument, pytest, pytest-cov, scikit-learn, skl2onnx, wheel.